tests/console/yast2_samba.pm

```diff
@@ -183,11 +183,15 @@ sub setup_samba_startup {
         assert_screen 'yast2_samba-server_start-during-boot';
     }
     else {
-        change_service_configuration(
-            after_writing => {start         => 'alt-e'},
-            after_reboot  => {start_on_boot => 'alt-a'}
-        );
+        send_key 'alt-e';
+        send_key 'end';
+        send_key_until_needlematch 'yast2_ncurses_service_start_after_writing_conf', 'up', 5, 1;
+        send_key 'ret';
+        send_key 'alt-a';
+        send_key_until_needlematch 'yast2_ncurses_service_start_on_boot_after_reboot', 'up', 5, 1;
+        send_key 'ret';
```

Comment:
what is the difference from the change_service_configuration_step? https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/yast2_widget_utils.pm#L70. pretty much they do the exactly the same steps. If the subroutine does not work maybe you should investigate that. optimal this does not need to change. And in case that there is a problem with that subroutine it would not be nice to stay broken here.

Reply:
... This is the whole point. Did you read the description of this PR ??

Reply:
there is a condition for the `send_key "ret"`. the workaround should be in the https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/yast2_widget_utils.pm#L70.

Reply:
This is a quick and temporary fix, the pemanent fix is the subject of the ticket.